src/BloomBrowserUI/bookEdit/js/interIframeChannel.ts

```diff
@@ -129,7 +129,11 @@ class interIframeChannel {
     $.ajax(ajaxSettings)
   }
 
-    getPageWindow(): Window {
+  getPageWindow(): Window {
     return (<HTMLIFrameElement>document.getElementById('page')).contentWindow;
   }
+
+  getAccordionWindow(): Window {
+    return (<HTMLIFrameElement>document.getElementById('accordion')).contentWindow;
+  }
```

Comment:
Looks like this new function is the only real change to this file...everything else is just fixing indentation?